content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Stargazer"
+date: 2022-05-03
+slug: kubernetes-1-24-release-announcement
+---
+
+# Kubernetes 1.24
+
+**Authors**: [Kubernetes 1.24 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+
+We are excited to announce the release of Kubernetes 1.24, the first release of 2022!
+
+This release consists of 46 enhancements: fourteen enhancements have graduated to stable, 
+fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha. 
+Also, two features have been deprecated and two features have been removed.     
+
+## Major Themes
+
+### Dockershim Removed from kubelet
+
+After its deprecation in v1.20, the dockershim component has been removed from the kubelet. 
+From v1.24 onwards, you will need to either use one of the other [supported runtimes](/docs/setup/production-environment/container-runtimes/) (such as containerd or CRI-O) 
+or use cri-dockerd if you are relying on Docker Engine as your container runtime. 
+For more information about ensuring your cluster is ready for this removal, please 
+see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/). 
+
+### Beta APIs Off by Default 
+
+[New beta APIs will not be enabled in clusters by default](https://github.com/kubernetes/enhancements/issues/3136). 
+Existing beta APIs and new versions of existing beta APIs, will continue to be enabled by default.
+  
+### Signing Release Artifacts
+
+Release artifacts are [signed](https://github.com/kubernetes/enhancements/issues/3031) using [cosign](https://github.com/sigstore/cosign) 
+signatures
+and there is experimental support for [verifying image signatures](/docs/tasks/administer-cluster/verify-signed-images/). 
```

Comment:
Does this verify link go anywhere? I don't see it work on the deploy preview?

Reply:
This points to a doc within the dev 1.24 branch.